examples/ButtonGroup.main.tsx

```diff
@@ -14,16 +14,16 @@ import {
 export default () => {
   return (
     <ButtonGroup>
-      <IconButton onClick={() => {}}>
+      <IconButton onClick={() => {}} label='add'>
```

Comment:
can you change all these labels (and aria-labels) to be **Sentence case**?

Reply:
Done!